subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/AbstractJavaGroovyCompileAvoidanceIntegrationSpec.groovy

```diff
@@ -314,22 +300,7 @@ public class ToolImpl {
         sourceFile.text = """
             public class ToolImpl {
                 public Object s = "12";
-                public void execute() { String s = toString(); }
-            }
-"""
-
-        then:
-        succeeds ":b:${language.compileTaskName}"
-        executedAndNotSkipped ":a:${language.compileTaskName}"
-        skipped ":b:${language.compileTaskName}"
-
-        when:
-        // add static initializer and constructor
```

Comment:
@wolfs, do we actually care about the empty constructor usecase, or was it just an easily testable dummy change?

Reply:
Separated to Groovy & Java test suite (changed in different commits for reviewability)